packages/react/src/components/icon/types.ts

```
@@ -1,16 +1,16 @@
 import type { ComponentType, ElementType } from 'react'
 import type { ClassValue } from 'clsx'
-import type { CSSObject } from '@nex-ui/system'
+import type { CSSObject, Interpolation } from '@nex-ui/system'
 import type { IconVariants } from '../../theme/recipes'
-import type { OverrideProps, SxProp } from '../../types/utils'
+import type { OverrideProps } from '../../types/utils'
 
 export interface IconPropsOverrides {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)